BSP/Inc/BSP_ADC.h

```diff
@@ -0,0 +1,82 @@
+#ifndef __BSP_ADC_H
+#define __BSP_ADC_H
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include "common.h"
+#include "config.h"
+
+enum ADCHardware {Accelerator, Brake};
+
+/**
+ * @brief   Initialize the ADC module
+ * @param   None
+ * @return  None
+ */ 
+void BSP_ADC_Init(void);
+
+/**
+ * @brief   Provides the number of active ADC channels 
+ * @param   None
+ * @return  numberOfADCValues, the number of ADC channels connected to the ADC providing raw ADC data
+ */ 
+int BSP_ADC_Number_Values();
+
+/**
+ * @brief   Provides pointer to array with the ADC values of csv file
+ * @param   None
+ * @return  rawADCValues, pointer to array that contains the ADC values in the array
+ */ 
+int* BSP_ADC_Read_Values();
```

Comment:
I don't want to be able to get an array of unlabeled values. It's better if I explicitly say which channel I want to get the data for, and then receive that channel's data specifically (especially since we made that enum). So, this function shouldn't be needed either.